code/modules/mob/living/carbon/carbon.dm

```diff
@@ -683,6 +683,9 @@
 	if (!..())
 		return FALSE
 
+	if (src.unslippable) //if unslippable, don't even bother making checks fam just don't slip me ok
```

Comment:
avoid using src when not necessary

also
>fam

Reply:
What about the fam? What's up with `src`? It's more verbose sure but it adds clarity.

Reply:
As a general rule we try to avoid having method params share a name with object vars in the first place, it's just in our style standards to avoid unnecessary srcs (and normally to prune them where they appear)

Reply:
"freestyle rapper" - man there's a ton of meme/funny/ironic comments in the code, I don't see why this is a problem.